docs/changes.rst

```diff
@@ -175,6 +175,9 @@ Other changes:
 - make RPC protocol more extensible: use named parameters.
 - RemoteRepository: misc. code cleanups / refactors
 - clarify cache/repository README file
+- the --keep-tag-files option has been deprecated in favor of --keep-exclude-tags.
+  additionally, both options now also accept directories as tags for backup
+  exclusion, #1999
```

Comment:
this should first talk about --exclude-if-present now just checking for the existance of a filesystem object of that name (not requiring the tag to be a file, but also accepting e.g. a directory).
the keeping behaviour should be described afterwards.